src/utils/resolveFilePath.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as os from 'os';
+import * as path from 'path';
+import { WorkspaceFolder } from 'vscode';
+
+/**
+ * Given a file path, tries to determine an absolute path. If relative, path is resolved relative to workspace folder.
+ * @param filePath The file path to resolve
+ * @param folder The workspace folder
+ */
+export function resolveFilePath(filePath: string, folder: WorkspaceFolder): string {
+    let replacedPath = filePath.replace(/\$\{workspaceFolder\}/gi, folder.uri.fsPath);
+
+    if (os.platform() === 'win32') {

```

Comment:
Is this stuff necessary? I thought `path.resolve` did some normalization of the separators

Reply:
I thought so too! But it doesn't :(